src/main/java/storage/StorageFile.java

```diff
@@ -0,0 +1,69 @@
+package storage;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.MismatchedInputException;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
+import data.TaskList;
+
+public class StorageFile {
+
+    private static final String workingDirPath = System.getProperty("user.dir");
+    private static final String DEFAULT_STORAGE_FILE_PATH = java.nio.file.Paths
+        .get(workingDirPath, "saveFile.json")
+        .toString();
+    private final ObjectMapper mapper = JsonMapper.builder()
```

Comment:
Perhaps a more descriptive variable name for `mapper` to describe what it maps and what it maps to?